docs/content/en/docs/guidelines-for-contributing.md

```diff
@@ -0,0 +1,35 @@
+## Guidelines for contributing
+
+* Always fork the repository then clone that fork to your local system
+  rather than clone `main` directly.
+  Keptn, like most open source projects,
+  severely restricts who can push changes directly to the `main` branch
+  to protect the integrity of the repository.
+* Keep your language clean and crisp.
```

Comment:
This is a docs-only guideline.  This list should be for both software and docs.  For guidelines specific to either software or docs, I think I like putting those right after this list but we could instead put them in the software and docs sections.  What do you think?